migaku-subtitle-syncer.py

```diff
@@ -62,3 +63,26 @@ def check_if_video_file(filename):
     parser = make_parser()
     args = parser.parse_args(args=unparsed_args)
     result = run(args)
+
+question = QMessageBox.question(
+    None,
+    'Save without ".synced"',
+    """
+Would you like to override the original subtitles, leave them as-is or discard all resynced subtitles?
```

Comment:
Change text to
```
Would you like to override the original subtitles, leave them as-is or discard all synced subtitles?

Note: this will overwrite all existing subtitles with the same name!

Save - Replaces each original subtitle with its synced counterpart
Close - Quit as-is without renaming subtitles further
Discard - Remove synced subtitles (in case of failure)
```